app/controllers/todos_controller.rb

```diff
@@ -1,9 +1,12 @@
 class TodosController < ApplicationController
   before_action :set_todo, except: [:new, :index, :create]

```

Comment:
as per Kevin's suggestion, maybe 'only' should be preferred over 'except'. I think doing it this way serves as a good semantic reminder that we're whitelisting only what's necessary.